backend/services/scheduler_service.py

```
@@ -42,15 +45,18 @@ def __init__(self):
         self.repository = get_repository()
         
         # 스케줄러 설정
-        self.interval_minutes = 5  # 기본 1시간 간격
+        self.interval_minutes = settings.scheduler_interval_minutes
         self.auto_start = True  # 자동 시작 여부
     
-    async def start(self, interval_minutes: int = 60):
+    async def start(self, interval_minutes: int = None):
```

Ruff (0.13.1)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)